x/axelarnet/keeper/msg_server.go

```diff
@@ -176,7 +176,7 @@ func (s msgServer) ExecutePendingTransfers(c context.Context, _ *types.ExecutePe
 	}
 
 	for _, pendingTransfer := range pendingTransfers {
-		recipient, err := sdk.AccAddressFromBech32(pendingTransfer.Recipient.Address)
+		recipient, err := funcs.Must(types.ToCosmosAddress(pendingTransfer.Recipient.Address)).AccAddress()
```

Comment:
Don't think i'm a fan of putting metadata in the address to indicate a wasm recipient, since this address is being used in a few places already by the time it's translated in here and allows multiple representation for the same address.

Shall we add `wasm/` as a prefix to the payload? This is checked only if the destination is a cosmos chain. For consensus messages, the prefix can be `native/` or something?